app/services/slack_service.py

```
@@ -18,13 +18,31 @@ class SlackService:
+        return db.query(SlackConnection).filter(SlackConnection.user_id == user_uuid).first()
+
     @staticmethod
     def create_slack_connection(db: Session, user_id: str, slack_data: SlackConnectionCreate) -> SlackConnection:
         """Create or update Slack connection"""
         
+        
+        try:
+            if isinstance(user_id, str):
+                user_uuid = uuid.UUID(user_id)
+            else:
+                user_uuid = user_id
```

Ruff (0.11.9)

Use ternary operator `user_uuid = uuid.UUID(user_id) if isinstance(user_id, str) else user_id` instead of `if`-`else`-block

Replace `if`-`else`-block with `user_uuid = uuid.UUID(user_id) if isinstance(user_id, str) else user_id`

(SIM108)